### ISSUE TITLE
Add all settings to Foundation if Foundation exists

### ISSUE THREAD

**Issue by zackkatz** [GFExcel/gf-entries-in-excel#154]
If Foundation exists, the settings should be in a GravityExport tab (not a separate GravityExport Lite tab).

**Comment by doekenorg** [comment]
@zackkatz You say 

> If Foundation exists

But that's on us, right? Wouldn't Foundation be included in the next major? Or is that not shipped with the plugin? 

If it is, I think we should port the settings to Foundation for consistency. If it isn't; then we should keep it on the plugin, to avoid having multiple places.

**Comment by zackkatz** [comment]
We haven't prepared Foundation for .org. It will take some amount of work to make it compliant.

**Comment by doekenorg** [comment]
Right, so In that case; I guess we could check for Foundation; and duplicate the settings over there; whilst saving them to the plugin for BC.

**Comment by mrcasual** [comment]
Another option is to integrate Foundation but disable products/licensing if it's just GravityExport Lite that's using it.

**Comment by doekenorg** [comment]
> Another option is to integrate Foundation but disable products/licensing if it's just GravityExport Lite that's using it.

I like this option best TBH. Since we are moving to 2.0, this is the best time to introduce a new settings page + location where the settings are stored.

**Comment by zackkatz** [comment]
@mrcasual we would need opt-in metrics reporting and Beacon
